.github/pr-commenter.yml

```diff
@@ -51,6 +51,8 @@ comment:
         - '*'
         - '**/*'
       body: |
+        - ✍️ If your PR fully fixes an issue, please include the text `Fixes #issue_no` in any line of the PR description. This will make the issue be automatically be closed when the PR is merged. If your PR is related to an existing issue but does not fix it completely, please link the issue anywhere in the description of the PR with `#issue_no`. You can read more about this in [Github: Linking a pull request to an issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
```

Comment:
```suggestion
        - 🔗 If your PR fully fixes an issue, please include the text `Fixes #issue_no` in any line of the PR description. This will make the issue be automatically be closed when the PR is merged. If your PR is related to an existing issue but does not fix it completely, please link the issue anywhere in the description of the PR with `#issue_no`. You can read more about this in [Github: Linking a pull request to an issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
```